bin/sandbox.sh

```diff
@@ -35,22 +35,29 @@ fi
 
 cd ./sandbox
 
+# Tags for below gems were made on 1 August 2022
 if [ "$SPREE_AUTH_DEVISE_PATH" != "" ]; then
   SPREE_AUTH_DEVISE_GEM="gem 'spree_auth_devise', path: '$SPREE_AUTH_DEVISE_PATH'"
 else
-  SPREE_AUTH_DEVISE_GEM="gem 'spree_auth_devise', github: 'spree/spree_auth_devise', branch: 'main'"
+  # For now tag on branch main will be used,
+  # in the future replace it with 4.5-stable release
+  SPREE_AUTH_DEVISE_GEM="gem 'spree_auth_devise', github: 'spree/spree_auth_devise', tag: 'v4.5.0-development'"
 fi
 
 if [ "$SPREE_GATEWAY_PATH" != "" ]; then
   SPREE_GATEWAY_GEM="gem 'spree_gateway', path: '$SPREE_GATEWAY_PATH'"
 else
-  SPREE_GATEWAY_GEM="gem 'spree_gateway', github: 'spree/spree_gateway', branch: 'main'"
+  # spree_gateway 'main' branch has latest NEEDED changes so, for now tag will be
```

Comment:
"... has latest NEEDED changes - this tag needs to be replaced after release of 4.5-stable" sounds better in my opinion.